app/modules/enduser_setup.c

```diff
@@ -755,12 +755,20 @@ static int enduser_setup_write_file_with_extra_configuration_data(char *form_bod
     return 1;
   }
 
-  // write all key pairs as KEY="VALUE"\n
+  // write all key pairs as KEY="VALUE"\n into a Lua table, example:
+  // local p = {}
+  // p.wifi_ssid="ssid"
+  // p.wifi_password="password"
+  // p.device_name="foo-node"
+  // return p
+  vfs_write(p_file, "local p = {}\n", 13);
```

Comment:
Ah darn, I prepared exactly the same fix (except for the comment)
I then removed the spaces around '=' to have a common coding style throughout the file. :-)
Didn`t have a chance to check it though and so did not dare to push it.

Reply:
Oh, and I thought you had lost interest 😉 Good point about the spaces; now fixed.

Reply:
I was busy with #2799 and just had seen that the branch was in the main repo and thus shared.

Reply:
That's great, thanks. Could you please assign the issue to yourself then? Helps in trying to avoid that several people work on stuff in parallel.